Kentico/xperience-by-kentico-sitefinity-migration-tool
# Feature/media import

# Motivation

Which issue does this fix? Fixes #4

## Checklist


## How to test

If manual testing is required, what are the steps?

Run the console app to import media files from Sitefinity over to XbyK. You will need to configure the WebApplicationPhysicalPath setting to ensure UMT knows where to place the files.
